src/smt/set_defaults.cpp

```diff
@@ -581,6 +581,12 @@ void setDefaults(LogicInfo& logic, bool isInternalSubsolver)
     logic = log;
     logic.lock();
   }
+  if (options::bvAbstraction())
+  {
+    // bv abstraction may require UF
+    Notice() << "Enabling UF because bvAbstraction requires it." << std::endl;
+    needsUf = true;
```

Comment:
UF was only enabled if we use `bitblastmode::LAZY`, not it is always enabled with `bvAbstraction`.

Reply:
The latter I believe is implied by the former. @mpreiner @aniemetz is this right?

Reply:
That's right `bvAbstraction` is only used by `--bv-solver=lazy`.

Reply:
And will thus be removed.